packages/shared/src/components/cards/PostTags.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement, useMemo, useRef } from 'react';
+import { Post } from '../../graphql/posts';
+import Classed from '../../lib/classed';
+
+type PostTagsProps = Pick<Post, 'tags'>;
+
+const Chip = Classed(
+  'div',
+  'rounded-8 border border-border-subtlest-tertiary px-2 h-6 flex items-center justify-center typo-footnote text-text-quaternary my-2',
+);
+
+export default function PostTags({ tags }: PostTagsProps): ReactElement {
+  const totalLength = useRef(0);
```

Comment:
Why do you need ref when you are not setting it anywhere else? It can be just local var in `useMemo` right?

Reply:
Also can we maybe just do `slice(0, 15)` and then `map` it would save iterations and make it simpler I think.

Reply:
The slice is dynamic it's not one piece, so it can either be 2 or 3 or 4 tags.
I had the ref as let first, but eslint was complaining ref would be better

Reply:
> he slice is dynamic it's not one piece

Sorry, I don't understand? 

> I had the ref as let first, but eslint was complaining ref would be better

Probably because you had it outside of `useMemo`